docs/api/migration-guides/qiskit-1.0-features.mdx

```diff
@@ -8,6 +8,57 @@ description: Adapt to feature changes with Qiskit 1.0
 This guide describes migration paths for the most important feature changes in Qiskit&reg;  1.0, organized by module.
 Use the table of contents on the right side to navigate to the module you are interested in.
 
+## Qiskit 1.0 migration tool
+
+To ease the migration process, we've created a tool that detects removed import
```

Comment:
Instead of saying "we created this tool" I think it would be better to talk about this tool in the context of the Qiskit Ecosystem, what do you think? Saying "we created this thing" implies it is more of an IBM product rather than an open-source community focused project.

Side note: if its not in the ecosystem yet I think it should be added1

Reply:
Thanks, I've changed the wording to avoid it sounding like it's an official IBM product

> To ease the migration process, you can use the [`flake8-qiskit-migration`](https://github.com/qiskit-community/flake8-qiskit-migration) tool to detect removed import paths in your code and suggest alternatives.